src/faction_camp.cpp

```diff
@@ -149,660 +161,587 @@ void talk_function::camp_missions( mission_data &mission_key, npc &p )
+                                  "Skill used: survival\n"
+                                  "Difficulty: 1\n"
+                                  "Effects:\n"
+                                  "> Good for returning equipment you left in the hide site "
+                                  "shelter.\n"
+                                  "> Gear is left unattended and could be stolen.\n"
+                                  "> Time dependent on weight of equipment being sent forward or "
+                                  "back.\n \n"
+                                  "Risk: Medium\n"
+                                  "Time: 1 Hour Base + Travel\n"
+                                  "Positions: %d/1\n" ), npc_list.size() );
+        mission_key.add_start( "Relay Hide Site", _( "Relay Hide Site" ), "", entry,
+                               npc_list.empty() );
         if( !npc_list.empty() ) {
             entry = _( "Transfering gear to a hide site...\n" );
-            bool avail = false;
-            for( auto &elem : npc_list ) {
-                avail |= update_time_left( entry, elem );
-            }
-            entry = entry + _( "\n \nDo you wish to bring your allies back into your party?\n" );
-            mission_key.text["Recover Hide Relay"] = entry;
-            mission_key.push( "Recover Hide Relay", _( "Recover Hide Relay" ), "", true, avail );
+            bool avail = update_time_left( entry, npc_list );
+            mission_key.add_return( "Recover Hide Relay", _( "Recover Hide Relay" ), "", entry,
+                                    avail );
         }
     }
 
-    if( bcp->has_level( camp_ctr, 8, base_dir ) ) {
-        std::vector<std::shared_ptr<npc>> npc_list = companion_list( p, "_faction_camp_foraging" );
-        mission_key.text["Camp Forage"] = string_format( _( "Notes:\n"
-                                          "Send a companion to edible plans.\n \n"
-                                          "Skill used: survival\n"
-                                          "Difficulty: N/A \n"
-                                          "Foraging Possibilities:\n"
-                                          "> wild vegetables\n"
-                                          "> fruits and nuts dependening on season\n"
-                                          "May produce less food than consumed!\n"
-                                          "Risk: Very Low\n"
-                                          "Time: 4 Hours, Repeated\n"
-                                          "Positions: %d/3\n" ), npc_list.size() );
-        mission_key.push( "Camp Forage", _( "Forage for plants" ), "", false, npc_list.size() < 3 );
+    if( bcp->has_level( camp_ctr, 2, base_dir ) ) {
+        comp_list npc_list = companion_list( p, "_faction_camp_foraging" );
+        entry = string_format( _( "Notes:\n"
+                                  "Send a companion to edible plans.\n \n"
```

Comment:
What kind of edible plans are these? I'm picturing a russian superspy consuming the microfiche before james bond can stop him.

You could change it to `"Send a companion to gather edible plants.\n \n"` but I'd be sad about the loss of james bond references.

Reply:
Fixed.